api/net/Keycloak/CssHelper.cs

```diff
@@ -203,7 +203,7 @@ private async Task AddOrUpdateUserAsync(string username, Entities.User? user, TN
     public async Task<string[]> UpdateUserRolesAsync(string username, string[] roles)
     {
         var userRoles = await _cssService.GetRolesForUserAsync(username);
-        if (userRoles.Users.Length == 0) return Array.Empty<string>();
+        if (userRoles.Users.Length == 0) userRoles = new UserRoleResponseModel() { };
```

Comment:
This resolve an issue where roles were not being applied to a new user.  Keycloak Gold realm doesn't return anything if the user hasn't already been assigned a role.  Which is annoying and makes adding new users more difficult.